code/modules/shuttle/mini_dropship.dm

```diff
@@ -273,3 +280,40 @@
 	origin.shuttle_port.set_mode(SHUTTLE_CALL)
 	origin.last_valid_ground_port = origin.my_port
 	SSshuttle.moveShuttleToDock(origin.shuttleId, origin.my_port, TRUE)
+
+/obj/machinery/computer/camera_advanced/shuttle_docker/minidropship/proc/rappel_targeting(datum/source, atom/target, turf/location)
+	rappelsys = GLOB.rappel_systems
+	var/area/A = get_area(target)
+	var/turf/T = get_turf(target)
+	if(!A)
+		return
+
+	if(!T)
+		return
+
+	if(target.z == rappelsys.z)
+		to_chat(source, span_warning("That target is in space!"))
+		return
```

Comment:
this is not how to check if a target is in space check that the Z is groundside instead